charts/opentelemetry-demo/values.yaml

```diff
@@ -610,6 +610,8 @@ opentelemetry-collector:
       otlp:
         protocols:
           http:
+            # Bind to 127.0.0.1 to allow for `kubectl port-forward` to work
+            endpoint: 127.0.0.1:4318
```

Comment:
Do we need to modify grpc's endpoint as well?

Reply:
No. We only ask people to port-forward the HTTP port so you can send telemetry to it via a web browser.